misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingOptions.java

```diff
@@ -30,15 +30,22 @@ public class FormattingOptions {
 
     private boolean lineWrapping;
 
+    private boolean optimizeImports;
+
     private ForceFormattingOptions forceFormattingOptions;
 
-    FormattingOptions(int tabSize, String wsCharacter, int columnLimit, boolean lineWrapping,
-                      ForceFormattingOptions forceFormattingOptions) {
+    private ImportFormattingOptions importFormattingOptions;
```

Comment:
Do we need to both of these new fields to configure import optimization option? `optimizeImports` flag seems redundant to me